src/pybind/mgr/nfs/export.py

```diff
@@ -368,8 +363,17 @@ def _update_export(self, cluster_id: str, export: Export,
         if need_nfs_service_restart:
             restart_nfs_service(self.mgr, export.cluster_id)
 
-    @export_cluster_checker
-    def create_export(self, addr: Optional[List[str]] = None, **kwargs: Any) -> Tuple[int, str, str]:
+    def _validate_cluster_id(self, cluster_id: str) -> None:
+        """Raise an exception is cluster_id is not valid."""
+        clusters = known_cluster_ids(self.mgr)
+        log.debug("checking for %r in known nfs clusters: %r",
+                  cluster_id, clusters)
+        if cluster_id not in clusters:
+            raise ErrorResponse("Cluster does not exist",
```

Comment:
logging the cluster_id on this error may be useful

Reply:
It'd be a small behavior change, but it seems reasonable, will do

Reply:
done